ScilifelabDataCentre/dds_web
# Get total number of projects

## Before submitting this PR

1. **Description:** Save number of total projects to the database, Reporting table.
2. **Jira task / GitHub issue:** DDS-1554
3. **How to test:** 
    1. In the dds_web container, run `flask db upgrade` and `flask db downgrade` to verify the migrations. 
    2. After one `flask db downgrade`, add a row manually to the Reporting table, and then run `flask db upgrade`. There should be no errors.
4. **Type of change:** [_Check the relevant boxes in the section below_](#what-type-of-changes-does-the-pr-contain)
5. **Add docstrings and comments to code**, _even if_ you personally think it's obvious.

## What _type of change(s)_ does the PR contain?



  - [x] Breaking: The command nameneeds to be updated in the daily cron job

## Checklist

- [Sprintlog](../SPRINTLOG.md)
- Rebase / Update / Merge _from_ base branch (the branch from which the current is forked)
- Blocking PRs
- PR to `master` branch

## Actions / Scans



- **Black**: Python code formatter. Does not execute. Only tests.
  Run `black .` locally to execute formatting.
- **Prettier**: General code formatter. Our use case: MD and yaml mainly.
  Run `npx prettier --write .` locally to execute formatting.
- **Yamllint**: Linting of yaml files.
- **Tests**: Pytest to verify that functionality works as expected.
- **CodeQL**: Scan for security vulnerabilities, bugs, errors
- **Trivy**: Security scanner
- **Snyk**: Security scanner


<a href="https://gitpod.io/#https://github.com/ScilifelabDataCentre/dds_web/pull/1418"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
